webserver/views/api/v1/core.py

```diff
@@ -227,19 +222,20 @@ def get_many_lowlevel():
 
     :query query_method: *Optional.* Determines which query method should be used
 
-      `single` will use the single-query method. No parameter or other value will revert
+      `single` will use the single-query method. No parameter or another value will revert
       to the multiple-query method
 
     :resheader Content-Type: *application/json*
     """
     # Query option allowed for testing purposes:
     if request.args.get("query_method") == "single":
         # Use single-query method
-        recording_details = get_data_for_multiple_recordings(db.data.load_many_low_level)
+        recordings = check_bad_request_for_multiple_recordings()
+        recording_details = db.data.load_many_low_level(recordings)
+        return jsonify(recording_details)    
```

Comment:
I've moved the calls to check_bad_request_for_multiple_recordings() and load_many_low_level(recordings) to make this process easier to follow. However, I've assumed that I should leave the pre-existing flow for the multiple query method as is for now?